src/interchange/src/avro.rs

```diff
@@ -1182,10 +1213,11 @@ impl DebeziumDecodeState {
                         .ok_or_else(|| anyhow!("\"row\" is not an integer"))?;
                         let pos = usize::try_from(pos_val)?;
                         let row = usize::try_from(row_val)?;
+                        // TODO(btv) Add LSN handling here too (OCF code path).
+                        // Better yet, just delete this and make it go through the same code path as Kafka
                         if !self.dedup.should_use_record(
                             &file_val,
-                            pos,
-                            row,
+                            RowCoordinates::MySql { pos, row },
```

Comment:
Just to make sure I understand, does this mean that OCF + Postgres won't currently work right now?

Reply:
If tb is outputting a `source` stanza, it won't work -- I need to look into this to see.

Reply:
Actually, nevermind. It will work "fine", but it won't perfrorm any deduplication.